package/run.sh

```diff
@@ -21,11 +36,19 @@ fi
 
 env "INSTALL_RKE2_ARTIFACT_PATH=${CATTLE_AGENT_EXECUTION_PWD}" installer.sh
 
+if check_target_mountpoint || check_target_ro; then
```

Comment:
I believe this should probably be up at about Line 22 or so, otherwise it will be set too late.